frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,182 @@
+      {text}
+    </div>
+  );
+};
+
+export const BundleBanner = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const mainImage = (
+    <img
+      src={womanInBanner400w.src}
+      srcSet={`${womanInBanner400w.src} 400w, ${womanInBanner768w.src} 768w, ${womanInBanner1280w.src} 1280w`}
+      alt=""
+      className={styles["main-image"]}
+    />
+  );
+
+  return (
+    <div className={styles["bundle-banner-wrapper"]}>
+      <div className={styles["first-section"]}>
+        <div className={styles["main-img-wrapper"]}>{mainImage}</div>
+        <div className={styles["float-features-wrapper"]}>
+          <FloatingFeatures
+            icon={bundleFloatOne.src}
+            text="bundle-feature-one"
+            position="feature-one"
+            vars={{
+              num_vpn_servers: "400",
+            }}
+          />
+          <FloatingFeatures
+            icon={bundleFloatTwo.src}
+            text="bundle-feature-two"
+            position="feature-two"
+            vars={{
+              num_vpn_countries: "30",
+            }}
+          />
+          <FloatingFeatures
+            icon={bundleFloatThree.src}
+            text="bundle-feature-three"
+            position="feature-three"
+          />
+        </div>
+      </div>
+      {isFlagActive(props.runtimeData, "bundle") &&
+        isBundleAvailableInCountry(props.runtimeData) && (
+          <div className={styles["second-section"]}>
+            <div className={styles["bundle-banner-description"]}>
+              <h2>
+                <Localized
+                  id={"bundle-banner-header"}
+                  elems={{
+                    "vpn-logo": <VpnWordmark />,
+                  }}
+                >
+                  <span className={styles["headline"]} />
+                </Localized>
+              </h2>
+              <h3>{l10n.getString("bundle-banner-subheader")}</h3>
+              <p>{l10n.getString("bundle-banner-body")}</p>
+              <Localized
+                id={"bundle-banner-1-year-plan"}
+                elems={{
+                  b: <b />,
+                }}
+              >
+                <p />
+              </Localized>
+              <div className={styles["pricing-logo-wrapper"]}>
+                <div className={styles["pricing-wrapper"]}>
+                  <Localized
+                    id={"bundle-price-monthly"}
+                    vars={{
+                      monthly_price: getBundlePrice(props.runtimeData),
+                    }}
+                    elems={{
+                      "monthly-price": <p className={styles["price"]} />,
+                    }}
+                  >
+                    <span />
+                  </Localized>
+                  <Localized
+                    id={"bundle-price-save-amount"}
+                    vars={{
+                      savings: "??%", // Design states 50%
+                      old_price: "$??", // Design states $11.99
```

Comment:
@Vinnl @codemist where is this price coming from and when are we going to replace it?

Reply:
Currently the design specs have fixed values. @Vinnl suggested not hardcoding the strings in case these values change.

Reply:
Yeah, I think Santiago was looking to get more clarity about what the final prices would be next week. :crossed_fingers: